### DISCUSSION TITLE
Help available

### DISCUSSION THREAD

**Discussion by jbogaardt** [casact/FASLR#27] [Ideas]
@genedan,

What a cool (and very ambitious) project.  It would be awesome to see it get off the ground.  

If you need a sounding board on ideas, I'd be happy to participate.  Also, don't hesitate to point out areas where `chainladder-python` falls short in real-world/production application.  I'd happily modify any rough edges to make it a good abstraction layer for this project.

**Comment by genedan** [comment]
@jbogaardt , your help would be most welcome! I will certainly have questions about chainladder as I move forward.
